tests/integration.rs

```diff
@@ -185,3 +188,26 @@ fn contents(harness: Harness) {
 	let link1 = std::fs::read_link(d.path().join("link1")).unwrap();
 	assert_eq!(&link1, Path::new("dir1/dir2/dir3/file2"));
 }
+
+#[rstest]
+fn statfs(harness: Harness) {
+	let d = &harness.d;
+	let sfs = nix::sys::statfs::statfs(d.path()).unwrap();
+
+	assert_eq!(sfs.blocks(), 15751);
```

Comment:
The kernel's UFS driver is the source of truth, so you should assert that fuse-ufs's statfs matches what the kernel does.  This is what the kernel reports for me:
```
f_version = 538182936
f_type= 53
f_flags= 0x1000
f_bsize = 4096
f_iosize= 32768
f_blocks= 15751
f_bfree= 15480
f_bavail= 14220
f_files= 8702
f_ffree= 8693
f_syncwrites= 2
f_asyncwrites= 1
f_syncreads= 9
f_asyncreads= 0
f_namemax= 255
f_fstypename= ufs
f_mntfromname= /dev/md0
f_mntonname= /mnt
```
Note that somes actually come from fusefs itself and not the specific fuse file system, so you don't need to assert for those.  They include f_iosize, f_type, and f_a?sync(writes|reads) .  If f_version ever changes, then the entire test will need to be revisited.  But that doesn't happen very often.

Reply:
nix' [Statfs](https://docs.rs/nix/latest/x86_64-unknown-freebsd/nix/sys/statfs/struct.Statfs.html) doesn't provide most of these values.
Should I do it with libc?

Reply:
I think Nix provides everything that comes from the actual file system.  The only things I see that it doesn't provide, on FreeBSD at least, are stuff like f_version, f_syncwrites, f_mntfromname, etc.  We don't need to test those.

Reply:
Ah I misunderstood, I thought you wanted me to test for f_version, f_iosize, etc.

Reply:
Yeah, we don't need those.  But you should still add assertions for f_namemax (Nix: maximum_name_length) and f_bavail(Nix: block_available).

Reply:
Done